local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Should we be putting the `local.settings.json` file in the repo, which generally isn't a best practice? 
It's pretty easy to regenerate it locally using `func init`.

Reply:
for now I vote no.  it's too easy for tools and users to put something private in there and check it in.